eos-copy-host-mali-driver

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+#
+# eos-copy-host-mali-driver: Copy the host's GL drivers into a flatpak local
+# extension directory.
+#
+# Copyright (C) 2017 Endless Mobile, Inc.
+# Authors:
+#  Philip Chimento <philip@endlessm.com>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+EXTENSIONS_DIR=/var/lib/flatpak/extension
+if test -d /var/endless-extra/flatpak; then
+    EXTENSIONS_DIR=/var/endless-extra/flatpak/extension
+fi
+
+DEST_DIR="$EXTENSIONS_DIR/org.freedesktop.Platform.GL.mali450/arm/1.6/lib"
+mkdir -p "$DEST_DIR"
+ln -s /usr/lib/arm-linux-gnueabihf/mali400-egl-x11 "$DEST_DIR"
```

Comment:
I was assuming that it would be OK because of https://github.com/flatpak/flatpak/blob/2082b508719999c4c7f2decbfd958943948c0b21/common/flatpak-dir.c#L6432, which eventually gets passed to `flatpak_extension_new()` in https://github.com/flatpak/flatpak/blob/e40c736b3bb0b9d1eff6399642e02bcc27a09a9a/common/flatpak-utils.c#L3975. If a symlink is possible, I think it would be better, since then we really only have to run the service once ever. But, you may be right that the symlink ends up dangling because of being outside the sandbox...